examples/game_of_life/game_of_life.measui

```diff
@@ -33,11 +30,14 @@
 				<PlotRenderer AreaBaseline="[FillBaseline]Zero" BarBaseline="[FillBaseline]Zero" Id="a8ab81312dc44ba79aaa6f0bd4638311" PointFill="[SMSolidColorBrush]#ffa08bb0" PointShape="[PointShape]Rectangle" xmlns="http://www.ni.com/Controls.LabVIEW.Design" />
 				<PlotRenderer AreaBaseline="[FillBaseline]Zero" BarBaseline="[FillBaseline]Zero" Id="8e6d2058f5264da782653ac2e504d779" PointFill="[SMSolidColorBrush]#ff7f7f7f" PointShape="[PointShape]Diamond" xmlns="http://www.ni.com/Controls.LabVIEW.Design" />
 				<PlotRenderer AreaBaseline="[FillBaseline]Zero" BarBaseline="[FillBaseline]Zero" Id="31cae3c70d5a44bda76a92809b0df15c" PointFill="[SMSolidColorBrush]#ff91685c" PointShape="[PointShape]Cross" xmlns="http://www.ni.com/Controls.LabVIEW.Design" />
+				<HmiGraphPlot Channel="[string]{1ba96edd-78a7-455d-81ef-5baf3f090c83}/Output/game_of_life" HorizontalScale="[UIModel]47588ac02a284a3eb9b22b58b332203f" Id="d8135274d68e4040ba9e73dafcd964bc" IsDefaultPlot="[bool]False" Label="[string]game_of_life" VerticalScale="[UIModel]3723e2d0c993443693f33dfe4056930b">
+					<PlotRenderer Id="9b18f852f145411e9d6a8310e5de68e1" LineThickness="[double]1" PointFill="[SMSolidColorBrush]#ff865ea8" xmlns="http://www.ni.com/Controls.LabVIEW.Design" />
+				</HmiGraphPlot>
 			</ArrayGraph>
-			<HmiChartPlotLegend Graph="[UIModel]a195c6ada9394c68b3f01238d59258e0" Height="[float]28" Id="be4c8c11f02142d99cedc899cf7161d8" Left="[float]967" LegendHeightFollowsGraph="[bool]False" Top="[float]56" />
-			<HmiChartCursorLegend Graph="[UIModel]a195c6ada9394c68b3f01238d59258e0" Height="[float]80" Id="c3f85f09183b4fb18779f63278f331eb" Left="[float]200" MinHeight="[float]80" Top="[float]691" Visible="[bool]False" Width="[float]316" />
-			<HmiChartScaleLegend Graph="[UIModel]a195c6ada9394c68b3f01238d59258e0" Height="[float]52" Id="4767b1b23ed14aafa34af2b397b768d0" Left="[float]885" Top="[float]669" Visible="[bool]False" />
-			<ArrayGraphTools BackgroundColor="[SMSolidColorBrush]#fff7f7f7" Graph="[UIModel]a195c6ada9394c68b3f01238d59258e0" Height="[float]26" Id="eb446917a8b3480f91844fd1a7dc6abc" Left="[float]806" OffsetX="[float]611" OffsetY="[float]0" Top="[float]38" Width="[float]122">
+			<HmiChartPlotLegend Graph="[UIModel]a195c6ada9394c68b3f01238d59258e0" Height="[float]28" Id="be4c8c11f02142d99cedc899cf7161d8" Left="[float]788" LegendHeightFollowsGraph="[bool]False" Top="[float]63" Visible="[bool]False" />
```

Comment:
The top margin above the graph is larger than above the the controls on the left.

Reply:
I wasn't quite sure what to align, but I aligned the top of the label with the top of the graph controls.
![image](https://github.com/ni/measurementlink-python/assets/38357562/48d60360-3131-4e86-961a-35cfeb43ddfb)